src/Confluent.SchemaRegistry/Rest/Authentication/BearerHttpAuthenticator.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright 2016-2018 Confluent Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Refer to LICENSE for more information.
+
+using System.Net.Http.Headers;
+
+namespace Confluent.SchemaRegistry
+{
+    /// <summary>
+    ///     A HTTP authenticator implementing the 'Bearer' scheme
+    ///
+    ///     See: https://datatracker.ietf.org/doc/html/rfc6750
+    /// </summary>
+    public class BearerHttpAuthenticator : IHttpAuthenticator
```

Comment:
we don't want to include this specific case because it relates to a setup that we don't document or talk about.

Reply:
i mean we don't want this file.